backend/config.ts

```diff
@@ -85,7 +85,7 @@ export default {
   },
 
   redis: {
-    host: get(process.env.REDIS_HOST, 'localhost', requiredInProduction),
+    host: get('REDIS_HOST', 'localhost', requiredInProduction),
```

Comment:
The get method adds in process.env so it would've previously been trying to get process.env[process.env.REDIS_HOST